vignettes/reconnect.Rmd

```diff
@@ -0,0 +1,67 @@
+---
+title: "Disconnected, Reconnect and Reload"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Disconnected, Reconnect and Reload}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Custom disconnected screen
+
+Nobody likes it when an app stops responding. Traditionally, Shiny apps will be "grayed out". Since this doesn't provide a nice user experience, `{shinyMobile}` adds a custom disconnected screen. This screen (a toast) is displayed when the app is disconnected from the server.
+
+TODO: shinylive example
+
+```{r, eval = FALSE}
+library(shiny)
```

Comment:
@hypebright: Maybe we can add a bit more to this app such as some input buttons to show that when we reconnect, we try to keep the old state. In the current state, we can't see the difference.